pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-parent</artifactId>
+		<version>1.3.1</version>
+	</parent>
+
+	<artifactId>disqualified-officers-search-consumer</artifactId>
+	<version>unversioned</version>
+	<name>disqualified-officers-search-consumer</name>
+	<description>Disqualified officers search consumer</description>
+
+	<properties>
+		<java.version>11</java.version>
+		<spring.boot.version>2.6.4</spring.boot.version>
+
+		<main.class>uk.gov.companieshouse.disqualifiedofficers.search.DisqualifiedOfficersSearchConsumerApplication</main.class>
+
+		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+		<kafka-models.version>1.0.27</kafka-models.version>
+		<ch-kafka.version>1.4.4</ch-kafka.version>
+
+		<!--sonar configuration-->
+		<sonar.coverage.jacoco.xmlReportPaths>${project.basedir}/target/site/jacoco/jacoco.xml,
+			${project.basedir}/target/site/jacoco-it/jacoco.xml
+		</sonar.coverage.jacoco.xmlReportPaths>
+		<sonar.jacoco.reports>${project.basedir}/target/site</sonar.jacoco.reports>
```

Comment:
Do we need a `jacoco-maven-plugin` too like in https://github.com/companieshouse/company-profile-api/blob/main/pom.xml?

Reply:
cheers Vaughan, added